Chorelord/ChoreListController.swift

```diff
@@ -8,21 +8,60 @@
 
 import UIKit
 
+let choresName = ["Water Roof Plants", "Clean Sink, Coffee area & Countertops", "Change bathroom & kitchen towels / Stock TP", "Chorelord", "Take out kitchen & office trash", "Garage Cleanliness / Break down boxes", "Water Roof Plants", "Conference, Bathroom & Roof trash", "Front of office: sweep, pick up trash, water plants", "Empty Dishwasher", "Clean out Fridge, Clear kitchen clutter (put away snacks, etc)", "Office Clutter, Clean Conference Room", "Sweep the office / kitchen", "Take out compost"]
+
+let names = ["Baylee", "Alex", "Anya", "Ellie", "Lee", "Dean", "Tony", "Rudy", "Tim", "Evan", "America", "Geoff", "Mike", "Jon"]
+
 class ChoreListController: UITableViewController {
-  let chores = ["chore1", "chore2"]
   
   override func viewDidLoad() {
     super.viewDidLoad()
+    self.reorderList(names, start: "2016-02-01", end: "2016-02-15", interval: 7) // this is doing nothing right now, just for testing purposes
```

Comment:
You don't actually need `self.` here, since you're calling a member function from another member function.